v2/client/client.go

```diff
@@ -220,8 +220,8 @@ func (c *ceClient) StartReceiver(ctx context.Context, fn interface{}) error {
 			continue
 		}
 
-		if err := c.invoker.Invoke(ctx, msg, respFn); err != nil {
-			return err
-		}
+		err := c.invoker.Invoke(ctx, msg, respFn)
+		fmt.Println("this, however, is still nil", err)
```

Comment:
Because returning an error here is **fatal**, it means the receiver has to exit. You don't want a bad input to crash the receiver, that could be abused by any client.